pkg/info/server_info.go

```diff
@@ -109,3 +109,11 @@ func Read(opts ...Option) (*ServerInfo, error) {
 	}
 	return info, nil
 }
+
+// GetDefaultServerInfo returns a ServerInfo object with the default fields populated for Go-SDK
+func GetDefaultServerInfo() *ServerInfo {
+	serverInfo := &ServerInfo{Protocol: UDS, Language: Go, MinimumNumaflowVersion: MinimumNumaflowVersion,
+		Version: GetSDKVersion()}
```

Comment:
Now that we move GetSDKVersion to inside of GetDefaultServerInfo, can we make GetSDKVersion pkg private?